cms/djangoapps/contentstore/views/preview.py

```diff
@@ -299,8 +299,14 @@ def _studio_wrap_xblock(xblock, view, frag, context, display_name_only=False):
         if selected_groups_label:
             selected_groups_label = _('Access restricted to: {list_of_groups}').format(list_of_groups=selected_groups_label)  # lint-amnesty, pylint: disable=line-too-long
         course = modulestore().get_course(xblock.location.course_key)
-        can_edit = context.get('can_edit', True)
-        can_add = context.get('can_add', True)
+
+        if root_xblock and root_xblock.upstream and str(root_xblock.upstream).startswith('lct:'):
```

Comment:
We should be doing comparisons on the opaque key, not substring matches.

Reply:
Improved in ea131439e83ada72f5eddf439b0990069d99f0a2